### ISSUE TITLE
Enabling parser plugin: 'classProperties' [still not fixed]

### ISSUE THREAD

**Issue by badabam** [pedronauck/docz#652]
## Bug Report

I guess it's the same bug as Still https://github.com/pedronauck/docz/issues/295,
which was closed but never solved (see comments).

**Describe the bug**
Can't use experimental class properties, when defining a class in Playground

*A clear and concise description of what the bug is.*

Can't find a solution to enable classProperties
```
SyntaxError: This experimental syntax requires enabling the parser plugin: 'classProperties' (5:12)
```
I tried this
`doczrc.js`
```js
export default {
  classProperties: true, // not working
  modifyBabelRc: babelRc => ({
    ...babelRc,
    plugins: [
      ...babelRc.plugins,
      '@babel/plugin-proposal-class-properties', // neither working
    ],
  }),
}
```

**To Reproduce**
Create a new Playground like this:
```js
<Playground>
  () => {
    class WithState extends React.Component {
      state = { // this causes the error
        text: 'hi'
      }

      render() {
        return <div>{this.state.text}</div>
      }
    }
}
</Playground>
```

```shell
Failed to compile.

./src/components/index.mdx
SyntaxError: This experimental syntax requires enabling the parser plugin: 'classProperties' (5:12)
```

**Comment by sag1v** [comment]
Same here but with a different error:

> Module parse failed: Unexpected token
> You may need an appropriate loader to handle this file type.

I'm using this syntax in my library

**Comment by pedronauck** [comment]
Closed in order of #229
